src/boot/efi/meson.build

```diff
@@ -425,6 +425,7 @@ foreach tuple : [['systemd-boot@0@.@1@', systemd_boot_objects, false],
                            '-lgnuefi',
                            '-lgcc'],
                 install : tuple[2],
+                install_tag: 'sd-boot',
```

Comment:
the tag to be used should probably be specified in the array we iterate through further up, so that sd-stub and sd-boot can have two distinct tags